From 126e601e2276025f9a42ca83cb3f026665b235e9 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?P=C3=A1draig=20Brady?= <pbrady@redhat.com>
Date: Fri, 27 Jan 2012 17:58:17 +0000
Subject: [PATCH] fix `nova-manage image convert` exception

* bin/nova-manage (_convert_images): Don't convert
the image id to a number as it's now a uuid.

Change-Id: I4dfe5693d8f394fb9ee3f2c18c7e0b3f2524e931
---
 bin/nova-manage |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/bin/nova-manage b/bin/nova-manage
index 88b15d2..3fb9903 100755
--- a/bin/nova-manage
+++ b/bin/nova-manage
@@ -1965,8 +1965,7 @@ class ImageCommands(object):
                 with open(image_path) as ifile:
                     image = self.image_service.create(elevated, meta, ifile)
                 new = image['id']
-                print _("Image %(old)s converted to " \
-                        "%(new)s (%(new)08x).") % locals()
+                print _("Image %(old)s converted to %(new)s.") % locals()
             except Exception as exc:
                 print _("Failed to convert %(old)s: %(exc)s") % locals()
 
